docs/content/NewButton.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Buttons
+status: Alpha
+source: https://github.com/primer/react/tree/main/src/NewButton
+storybook: '/react/storybook?path=/story/composite-components-new-button--default-button'
+description: Button is core design component used for actions.
+---
+
+import {Props} from '../src/props'
+import {ComponentChecklist} from '../src/component-checklist'
+import {NewButton} from '@primer/components'
+
+## Usage
+
+### Installation
+
+```js
+import {NewButton} from '@primer/components/drafts'
```

Comment:
Can we export `Button` instead? it's scoped in drafts

Reply:
Well after migration this will be the only button and I need to update docs too at that time. Its okay if it stays this way right?